docs/src/cli/deploy-a-program.md

```diff
@@ -124,28 +124,83 @@ call to `deploy`.
 Deployment failures will print an error message specifying the seed phrase
 needed to recover the generated intermediate buffer's keypair:
 
-```bash
-=======================================================================
-To resume a failed deploy, recover the ephemeral keypair file with
-`solana-keygen recover` and the following 12-word seed phrase,
-then pass it as the [BUFFER_SIGNER] argument to `solana deploy` or `solana write-buffer`
-=======================================================================
-spy axis cream equip bonus daring muffin fish noise churn broken diesel
-=======================================================================
+```
+==================================================================================
+Recover the intermediate account's ephemeral keypair file with
+`solana-keygen recover` and the following 12-word seed phrase:
```

Comment:
The first time they `program deploy` they need to provide a signer (since the account is being created), but for resume you just need the address.  We could simplify this so they don't have to recreate the keypair for a resume

Reply:
I think that would be a really nice simplification!